timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TimeLockTestUtils.java

```diff
@@ -31,6 +31,8 @@
 import com.palantir.tritium.metrics.registry.DefaultTaggedMetricRegistry;
 
 public final class TimeLockTestUtils {
+    public static final String AGENT = "smith";
```

Comment:
I don't recommend making this public. Basically, we want every invocation to create a corresponding timelock cluster or whatever to return a separate namespaced instance.

Reply:
Should be good now?